lib/reporting/reports/suppliers/helpers/columns_helper.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module Reporting
+  module Reports
+    module Suppliers
+      module Helpers
+        module ColumnsHelper
+          include LineItemsAccessHelper
+
+          def producer
+            proc { |line_items| supplier(line_items).name }
+          end
+
+          def producer_address
+            proc { |line_items| supplier(line_items).address&.full_address }
+          end
+
+          def producer_abn_acn
+            proc do |line_items|
+              supplier = supplier(line_items)
+
+              supplier.abn.presence || supplier.acn
+            end
+          end
+
+          def email
+            proc { |line_items| supplier(line_items).email_address }
+          end
+
+          def hub
+            proc { |line_items| distributor(line_items).name }
+          end
+
+          def hub_address
+            proc { |line_items| distributor(line_items).address&.full_address }
+          end
+
+          def hub_contact_email
+            proc { |line_items| distributor(line_items).email_address }
+          end
+
+          def order_number
+            proc { |line_items| order(line_items).number }
```

Comment:
❓I could be wrong, but it looks like there could be multiple orders grouped together, so I don't think we should show an order number here, should we?

Reply:
Yes, you are right. Now, that I think of it and reviewed the [report](https://docs.google.com/spreadsheets/d/1XDi3whY2h2DnHQtYS0Fbogr_VTJmYhAXzt1p9ElBBbM/edit?gid=0#gid=0) again, each order and their respective order cycles should have been displayed here.

Reply:
Doing this fix now.

Reply:
I think we should display all of these order numbers for each line_item. It was missed before. 
Here's the fix: https://github.com/openfoodfoundation/openfoodnetwork/pull/12879/commits/8ee8b3714e8083d2c9a6aadd20163df08c364182

Reply:
Great! 
`±` Now I think the line item helpers are no longer needed: `variant`, `order`, `supplier`, `distributor`, `item_order_cycle`